presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1386,9 +1385,9 @@ private void printPlanNodesStatsAndCost(int indent, PlanNode... nodes)
 
         private Optional<String> formatPlanNodeStatsAndCost(PlanNode node)
         {
-            PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), UNKNOWN_STATS);
+            PlanNodeStatsEstimate stats = estimatedStatsAndCosts.getStats().getOrDefault(node.getId(), PlanNodeStatsEstimate.unknown());
             PlanNodeCostEstimate cost = estimatedStatsAndCosts.getCosts().getOrDefault(node.getId(), UNKNOWN_COST);
-            if (stats.equals(UNKNOWN_STATS) || cost.equals(UNKNOWN_COST)) {
+            if (stats.isOutputRowCountUnknown() || cost.equals(UNKNOWN_COST)) {
```

Comment:
Semantics change, should not be part of `Add PlanNodeStatsEstimate#unknown() method` refactor.

Reply:
Why don't you add `Do not print stats with unknown row count` commit as the _first_ commit (just with this change + introduction of `isOutputRowCountUnknown` method)?

Reply:
> Why don't you add Do not print stats with unknown row count commit as the first commit (just with this change + introduction of isOutputRowCountUnknown method)?

I asked you that question in the previous PR. I don't mind extracting it.